e2e/actions/actions.go

```diff
@@ -784,7 +785,11 @@ func (c actionTests) PersistentOverlay(t *testing.T) {
 		t.Fatalf("Unexpected error while running command.\n%s", res)
 	}
 
-	cmd = exec.Command("mkfs.ext3", "-q", "-F", ext3Img)
+	mkfsExt3Cmd, err := bin.FindBin("mkfs.ext3")
+	if err != nil {
+		t.Fatal("Unable to find 'mkfs.ext3' binary even though require.Command() was called")
```

Comment:
Probably worth outputting the error to aid troubleshooting (here and the other three similar spots)?
```suggestion
		t.Fatalf("Unable to find 'mkfs.ext3' binary even though require.Command() was called: %v", err)
```